pkg/cdi/cache.go

```diff
@@ -147,12 +146,12 @@ func (c *Cache) refresh() error {
 		devices    = map[string]*Device{}
 		conflicts  = map[string]struct{}{}
 		specErrors = map[string][]error{}
-		result     []error
+		errs       = []error{}
```

Comment:
```suggestion
		errs       error
```

Reply:
Yeah, actually I did this wrong as I flattened all errors to a single joinError. I updated it to return a joinError of one joinError per Spec errors.